specification/indices/_types/IndexSettings.ts

```diff
@@ -534,44 +534,15 @@ export class SlowlogTresholdLevels {
 }
 
 export class Storage {
-  type: StorageType
```

Comment:
I can see it here: https://github.com/elastic/elasticsearch/blob/1ad6dbc5991b77ce1b05d5015e1f5ac758e1fcb4/server/src/main/java/org/elasticsearch/index/IndexModule.java#L427-L432